### PR TITLE
Implement random shift button

### PR DESCRIPTION
**Pull request by MyIgel**

### CODE REVIEW

File: resources/views/pages/user-shifts.html [hunk 1]
```diff
@@ -47,7 +47,10 @@ <h1>%title% %add_link%</h1>
                     <button type="button" class="btn btn-secondary set-time" data-hours="8">%set_next_8h%</button>
                 </div>
                 <div class="btn-group mb-1" role="group">
-                    %buttons%
+                    %random%
```
Comment (xuwhite) [inline resources/views/pages/user-shifts.html:50]: could you please make a config option for that?

File: resources/lang/de_DE/default.po [hunk 2]
```diff
@@ -228,6 +228,9 @@ msgstr "Das Aufbau Start Datum muss vor dem Abbau Ende Datum liegen."
 msgid "Public Dashboard"
 msgstr "Öffentliches Dashboard"
 
+msgid "Random Shift"
```
Comment (xuwhite) [inline resources/lang/de_DE/default.po:231]: Pls use a translation key

File: resources/lang/en_US/additional.po [hunk 3]
```diff
@@ -250,6 +250,9 @@ msgstr ""
 "Since the deleted shift was already done, "
 "we added a worklog entry instead, to keep your work hours correct."
 
+msgid "notification.shift.no_next_found"
+msgstr "Next shift not found."
```
Comment (xuwhite) [inline resources/lang/en_US/additional.po:254]: I would prefer "There is no available shift."

File: resources/lang/de_DE/additional.po [hunk 4]
```diff
@@ -251,6 +251,9 @@ msgstr ""
 "Da die gelöschte Schicht bereits vergangen ist, "
 "haben wir einen entsprechenden Arbeitseinsatz hinzugefügt."
 
+msgid "notification.shift.no_next_found"
+msgstr "Es konnte keine nächste Schicht gefunden werden."
```
Comment (xuwhite) [inline resources/lang/de_DE/additional.po:255]:
```suggestion
msgstr "Es wurde keine verfügbare Schicht gefunden."
```

**Review by xuwhite — COMMENTED** [review]
some small suggestions

File: resources/views/pages/user-shifts.html (hunk 1 above)
Comment (MyIgel) [reply resources/views/pages/user-shifts.html:50]: I don't think that it would be useful for anyone to disable it?

Comment (MyIgel) [reply resources/views/pages/user-shifts.html:50]: I don't think that it would be useful for anyone to disable it?

**Review by xuwhite — APPROVED** [review]
Looks good to me

**Pull request merged**